auth/service_accounts.go

```diff
@@ -0,0 +1,4 @@
+package auth
+
+// OnlineRegistrationServiceAccount is the service account name of the online-registration app
+const OnlineRegistrationServiceAccount = "online-registration"
```

Comment:
if not here, we gotta put these somewhere in one place :)

Reply:
Right now we just hardcode the name of the sa. Later we will associate a set of scopes for all service accounts. And instead of checking the SA name we will check the scopes via our standard authorization workflow.

Reply:
Cool, removed - and hard-coded. 85ed444